### PR TITLE
Week 8 Article

### PR DESCRIPTION
**Pull request by indekkusu-era**
Storyboard Article

- Article Type: opinion
- Header: Media Centre Behind the Scene: The Essence of Storyboarding
- Subtitle: The storyboard is undeniably one of the essence in the astounding experiences in Tiebreaker Showcase, what are the techniques we used behind these effects?
- Author(s): Polytetral, IndexError

Grand Finals Highlights

- Article Type: highlight
- Header: The Grand Finale: China vs. Singapore
- Subtitle: What is the conclusion of 4 Digit MWC 2023?
- Author(s): Polytetral, IndexError

4DM23 The Grand Finalists Interview feat. Canada, China and Singapore

- Article Type: interview
- Header: 4DM23 The Grand Finalists Interview feat. Canada, China and Singapore
- Subtitle: What are the afterthoughts of China after winning, Singapore after trying their best to challenge them, and Canada who secured the third place of this tournament!
- Author(s): PorkIsGreat

4DM23 Host Interview

- Article Type: interview
- Header: 4DM23 Coordinators Interview: The thoughtful insight of the tournament organizations. feat. Logan, Danny and Paturages
- Subtitle: Exploring on how tournament organizers are doing, and how the tournament was made, as well as coordination inside the staff team.
- Author(s): PorkIsGreat

4DM23 Poolers Interview

- Article Type: interview
- Header: 4DM23 Poolers Interview: Exploring the mappool managements and custom maps with DemiFiendSMT, Eclipse_ and FelixSpade
- Subtitle: We again explore the thoughts behind pooling inside the Mappool Selectors team, as well as the thoughts inside the custom mappers.
- Author(s): PorkIsGreat

4DM23 Streamer, Referee and Commentator interview

- Article Type: interview
- Header: 4DM23 Referee, Streamer and Commentator Interview: What's behind the front stage of 4 Digit MWC 2023 Matchups? feat. Poity, Dynascape and NightNarumi
- Subtitle: Let's hear the thoughts of Referee, Streamer and Commentator on this tournament!
- Author(s): PorkIsGreat

### CODE REVIEW

**Comment by indekkusu-era** [comment]
oh yea note: imma need to add sb github repo in and write some introduction

File: highlights/gf.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Singapore pulled out the pro gamer move by picking the My Chemical Romance song, **Welcome To The Black Parade**, as the **Lofty's Ending**, the relentless speed stream map testing how much the players can carry on. The lead gradually goes to the Singapore side as they know how to carry on, and it became more explicit that Singapore will carry this point. And certainly, they took it away, with the astounding lead of 95738 points, taking a lead with their own pick again to 5-4.
```
Comment (polytetralhx) [inline highlights/gf.md]: 'pro gamer' sounds pretty weird, maybe something more relevant like an 'unexpected' pick would make more sense

**Review by polytetralhx — APPROVED** [review]
If possible, bolden the score differences and maps picked, while mentioning the map tag in the pool eg. RC3, LN2, etc.

Will wait for grammatical checks by Leon but otherwise okay for me

**Comment by Leon1805** [comment]
>

File: highlights/gf.md (hunk 1 above)
Comment (indekkusu-era) [reply highlights/gf.md]: alright, actually i will change to "another move" because it's not really that unexpected compared to other picks

**Comment by indekkusu-era** [comment]
updated descriptions for all articles

**Pull request merged**